longevity_test.py

```diff
@@ -142,6 +143,14 @@ def test_custom_time(self):
                 for stress in verify_queue:
                     self.verify_stress_thread(queue=stress)
 
+
+        # # Collect data about partitions and their rows amount
+        # validate_partitions = self.params.get('validate_partitions', default=None)
+        # if validate_partitions:
+        #     self.log.debug('Save partitons info before reads')
```

Comment:
do we need this commented out code?

Reply:
This is new part that was asked by Shlomi. I did it but didn't test still because of the test was changed so that this part is not run. I'll discuss with Roy about this

Reply:
Let's add this code back while we still testing this.
It'll save testing time later.